File: check_db.py
import sqlite3
db = sqlite3.connect("E:/Pagina_seiva/seiva-static/backend/data/database.sqlite")
r = db.execute("SELECT name FROM sqlite_master WHERE type='table' AND name='descuentos_cantidad'").fetchone()
print("EXISTS" if r else "MISSING")
if r:
    c = db.execute("SELECT COUNT(*) FROM descuentos_cantidad").fetchone()
    print("rows:", c[0])
db.close()

File: seiva-static/backend/check-db.js
const Database = require("better-sqlite3");
const db = new Database("./data/database.sqlite");
try {
  const r = db.prepare("SELECT name FROM sqlite_master WHERE type='table' AND name='descuentos_cantidad'").get();
  console.log(r ? "EXISTS" : "MISSING");
  if (r) {
    const c = db.prepare("SELECT COUNT(*) as c FROM descuentos_cantidad").get();
    console.log("rows:", c.c);
  }
} catch(e) {
  console.log("ERROR:", e.message);
}
db.close();
